docs/man/ntp.toml.5.md

```diff
@@ -133,6 +133,14 @@ sources.
     the number of seconds (i.e. two to the power of the interval). The default
     value of 4 results in an interval of 16 seconds.
 
+`ntp-version` = `4` | `5` | `"auto"` (**4**)
+:   Which NTP version to use for this source. By default this uses NTP version
+    4. You can use `5` to set the protocol version to the draft NTPv5
+    specification, but your source must support the same draft. To use the
+    experimental upgrade protocol set the NTP version to `"auto"`. This uses a
+    specially constructed NTPv4 packet to attempt an upgrade to the supported
+    draft NTPv5 version if the source also supports the same draft version.
```

Comment:
Perhaps we should add that NTPv5 support is still considered in beta, and that things might not be backwards compatible with future versions.

Reply:
Fixed in a3c075476f93ee85b1735790d2ac90df37f2d190